src/test/java/com/wikia/webdriver/common/core/elemnt/Wait.java

```diff
@@ -46,6 +49,38 @@ public WebElement forElementPresent(By by) {
   }
 
   /**
+   * Checks if the element is clickable on browser
+   * <p/>
+   * * @param element The element to be checked
+   */
+  public WebElement forElementClickable(WebElement element) {
+    changeImplicitWait(0, TimeUnit.MILLISECONDS);
+    try {
+      element.getTagName();
+    } catch (WebDriverException e) {
```

Comment:
@ludwikkazmierczak I copied this block from forVisibility method. Is this preventing staleelement reference, or they're doing something else ? It needs a comment